nemo/collections/asr/losses.py

```diff
@@ -41,11 +41,11 @@ def output_ports(self):
         # return {"loss": NeuralType(None)}
         return {"loss": NeuralType(elements_type=LossType())}
 
-    def __init__(self, num_classes):
+    def __init__(self, num_classes, zero_infinity=False):
```

Comment:
can you please add docstring to this class

Reply:
Sounds good.

Reply:
Fixed